vertx-client-shaded/build.gradle

```
@@ -83,6 +83,8 @@ shadowJar {
     }
 }
 
+test.dependsOn shadowJar
```

thanks for fixing this dependency! I have a small NIT:
```suggestion
tasks.named('test') {
    dependsOn 'shadowJar'
}
```